sensitivity.tex

```diff
@@ -0,0 +1,126 @@
+        \hline
+        \textbf{Parameter} & \textbf{Original} & \textbf{Values} \\
+        \hline
+        Lifetime of French \glspl{LWR} [years] & 60  & 65, 70, 80 \\
+        Breeding Ratio of \glspl{ASTRID} & 1.08 & 1.11, 1.15, 1.18 \\ 
+        \hline
+    \end{tabularx}
+    \caption {Two parameters are varied for the sensitivity analysis - the 
+              lifetime of French \glspl{LWR} and the breeding ratio of \glspl{ASTRID}.
+              The parameters are increased to analyze their impact on the reprocessing
+              demand for the transition.}
+    \label{tab:sen_par}
+\end{table}
+
+\subsection{Breeding Ratio}
+
+The result of the breeding ratio sensitivity analysis is shown in
+\cref{fig:br_rep}.
+Increase in the breeding ratio of \gls{ASTRID} reactors
+decreases the monthly average reprocessing, but the 
+average \gls{UOX} reprocessing demand doesn't dramatically change,
+because the time that \gls{ASTRID} fuel supplied from reprocessed
+\gls{UOX} decrease with increase breeding ratio. The average monthly reprocessing
+demands are calculated by dividing the total amount by the number of non-zero elements.
+
+\begin{figure}[htbp!]
+    \begin{center}
+        \includegraphics[scale=0.6]{./images/sensitivity/br.png}
+    \end{center}
+    \caption{Increasing the breeding ratio decreases the average monthly reprocessing for all fuels,
+             but the average monthly \gls{UOX} reprocessing does not decrease due to shorter supply time. }
+    \label{fig:br_rep}
+\end{figure}
+
+
+\begin{figure}[htbp!]
+    \begin{center}
+        \includegraphics[scale=0.6]{./images/sensitivity/br_uox.png}
+    \end{center}
+    \caption{Increasing the breeding ratio decreases the number of total \gls{UOX} \gls{UNF}
+             since the \gls{ASTRID} becomes self-sustaining earlier, no longer requiring
+             plutonium from reprocessing \gls{UOX}.}
+    \label{fig:eu_tail}
+\end{figure}
+
+
+\subsection{Lifetime Extension of French \glspl{LWR}}
+Extending the lifetime of French \glspl{LWR} dramatically lowers the average
+monthly \gls{UOX} reprocessing demand, since the \gls{ASTRID} deployment becomes less
+aggressive. the plutonium demand is delayed and allows the reprocessing plant more
+time to produce the same amount of plutonium for \gls{ASTRID} reactors. The
+extended deployment schemes are illustrated in figure \ref{fig:ext}. 
+
+\begin{figure}[htbp!]
+    \begin{center}
+        \includegraphics[scale=0.6]{./images/sensitivity/ext_uox.png}
+    \end{center}
+    \caption{Increasing the lifetime of French \glspl{LWR} decreases the monthly
+             average \gls{UOX} reprocessing demand.}
+    \label{fig:ext}
+\end{figure}
+
+
+\begin{figure}[htbp!]
+    \begin{center}
+        \includegraphics[scale=0.6]{./images/sensitivity/ext_all.png}
+    \end{center}
+    \caption{The shift in \gls{ASTRID} deployment has little impact on the total monthly
+             reprocessing capacity.}
```

Comment:
I think you mean demand, rather than capacity. If you want to say capacity, then you could say "required reprocessing capacity"